### ISSUE TITLE
django 2 compatibility

### ISSUE THREAD

**Issue by macolo** [Organice/djangocms-maps#18]
for example:

```
  |   File "/usr/local/lib/python3.7/site-packages/djangocms_maps/migrations/0001_initial.py", line 23, in Migration
web_1  |     auto_created=True)),
web_1  | TypeError: __init__() missing 1 required positional argument: 'on_delete'
```

**Comment by bittner** [comment]
Thank you for the report! Related resources:

* [Source code](https://github.com/Organice/djangocms-maps/blob/master/djangocms_maps/migrations/0001_initial.py#L17-L23)
* [Django release notes for 2.0](https://docs.djangoproject.com/en/3.0/releases/2.0/#features-removed-in-2-0)

Can you try to add a
```python
on_delete=models.CASCADE
```
keyword parameter, in the vein of [OneToOneField](https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.OneToOneField) and see if that works for you?

**Comment by macolo** [comment]
yes, that works, and it also needs to be added in any occurence of the OneToOneField in any of the existing migration files.

**Comment by bittner** [comment]
Which ones? The ones referred to by [dependencies](https://github.com/Organice/djangocms-maps/blob/master/djangocms_maps/migrations/0001_initial.py#L9-L11)? That could maybe be squashed as suggested by the [Django documentation](https://docs.djangoproject.com/en/3.0/topics/migrations/#migration-squashing).

**Comment by macolo** [comment]
Hi Peter, I quickly tested this in one of my projects and it doesnt throw any errors, and it appears that other projects have been made django2 compatible in the same way so I thought i'd contribute a small PR: https://github.com/Organice/djangocms-maps/pull/19

does this work for you?

**Comment by bittner** [comment]
Fixed with commit 8596eba410260dd3fdbbb53ac9e77cf29e271a8c via PR #19.

Thank you, @macolo!
